%YAML:1.0

BaseLidar:
  pcds_file: /media/lam_data/标定数据/上海小车/20号/直线往返/lidarpcd/23-12-28_12-24-44_826/pcd/lidar_roof_front
  max_dis: 40
  min_dis: 1
  # hesai64, bp32, helios32, ft120, yijing
  lidar_type: ft120 
  tz: 1.2

RefNum: 0
Step: 5
SceneNum: 3

DrPosesFile: /media/lam_data/标定数据/上海小车/20号/直线往返/video/23-12-28_11-28-06_902/video/dr_tum.txt

ImageFilesPath: /media/lam_data/标定数据/上海小车/20号/直线往返/video/23-12-28_11-28-06_902/jpg

CamsIntrinsicPath: /media/lam_data/标定数据/上海小车/20号/直线往返/camera_baselink/camera
CamsExtrinsicPath: /media/lam_data/标定数据/上海小车/20号/直线往返/camera_baselink/camera

init_extrinsics_file: /home/wd/datasets/3/init_transform_dr_front.pb.txt

CameraName:
  - avm_front
  - avm_left
  - avm_rear
  - avm_right
  # - front_near
  # - left_front
  # - left_rear
  # - rear_medium
  # - right_front
  # - right_rear

AroudCameraStampFile: /media/lam_data/标定数据/上海小车/20号/直线往返/video/23-12-28_11-28-06_902/video/video_frame_info

Canny.gray_threshold: 10
Canny.len_threshold: 50

# Params for Voxel Cutting & Plane Fitting & Edge Extraction
UseAdaVoxel: 0
Voxel.size: 0.6
# Voxel.size: 0.5
Voxel.eigen_ratio: 0.05
Plane.min_points_size: 30
Plane.normal_theta_min: 20
Plane.normal_theta_max: 170
Ransac.dis_threshold: 0.02
# Ransac.dis_threshold: 0.015
Edge.min_dis_threshold: 0.08
# Edge.min_dis_threshold: 0.05

save_lidar_extrinsic_name: roof_transform.pb.txt

ResultPath:  /home/wd/datasets/3/RESULT_2

